cli.js

```diff
@@ -4,7 +4,8 @@
 // calliope). This gives us everything that comes after those two, which is
 // what we care about.
 const args = process.argv.slice(2);
-const omitSetupReport = args.includes('--omit-setup-report');
+const isInit = args[0] === 'init';
+const omitSetupReport = args.includes('--omit-setup-report') || isInit;
```

Comment:
Changes to this file amount to: When running the `init` command, only print the Calliope version without reporting on all the usual reporting about what overrides, custom tasks, config, etc it has found.